extensions/panache/hibernate-orm-panache-kotlin/runtime/pom.xml

```diff
@@ -156,30 +156,34 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-javadoc-plugin</artifactId>
+                <version>3.1.1</version>
+                <configuration>
+                    <failOnError>false</failOnError>
+                </configuration>
+            </plugin>
             <plugin>
                 <groupId>org.jetbrains.dokka</groupId>
                 <artifactId>dokka-maven-plugin</artifactId>
-                <version>0.10.1</version>
+                <version>1.4.30</version>
```

Comment:
Does this have to be the same as the Kotlin version we use?

Reply:
That I don't know, maybe but I don't have the time to wait so it can be fixed later :).

Reply:
It does not. That's the latest version in central.